container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -338,6 +340,7 @@ will change values on each row under consideration. See the [Intro to Datasets](
   - if `x` is the empty string, return `null`
   - if `x` is a string that can be converted to a number, return the number
   - otherwise, return `x` unchanged
+- `hash(expr)` return the hash of the value in `expr`. See also [`rowHash()`](#rowHash).
```

Comment:
J'imagine que le "see also" me dit que la description du rowHash va s'appliquer aussi, mais je trouve pas ça méga clair. Si le hash a un nom, je le mettrais ici, genre "return the patate-poil hash ..." et je laisserais le see also quand même.